src/execution/StepExecutionManager.ts

```diff
@@ -62,6 +75,84 @@ export class StepExecutionManager {
     let process = statusManager.findOrCreateProcess(step, currentProcessType)
 
     if (process.status !== 'DONE') {
+      // TODO: update @lifi/types repo for these changes [DO NOT MERGE IF YOU SEE THIS]
+
+      console.log('NOT DONE YET!!')
+
+      console.log({ isSafeSigner, process, step })
+
+      if (isSafeSigner && step.execution.status === 'SAFE_SIGNING_PENDING') {
```

Comment:
Can we rename the status message to "MULTISIGNATURE_SIGNING_PENDING"?
that way its more general

Reply:
I don't like the idea of a safe wallet spread across all our repos. Is there a way to have it only within Jumper? What if we need another multisig wallet support tomorrow? We should stay wallet agnostic in the SDK.

Reply:
Changed back to PENDING